src/style/global-style.js

```diff
@@ -0,0 +1,26 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+  body {
+    color: rgba(0, 0, 0, 0.9);
+    font-family: 'Lato';
+    font-size: 14px;
+    margin: 0px;
+    padding: 0px;
+    text-rendering: optimizeLegibility;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  button, input, select, textarea {
+    font-family: inherit;
+  }
+
+  h1, .h1 { font-size: 24px; font-weight: 900; line-height: 32px; }
```

Comment:
Didn't we decide not to have the heading styles in here anymore?

Reply:
we can't change it for now as our storybook stories will change, which will trigger a few chromatic failures. I'll create a task for it to be removed in the future.